applications/mongodb/chart/values.yaml

```diff
@@ -11,13 +11,13 @@ mongodb:
     enabled: false
   containerSecurityContext:
     enabled: false
-#  resources:
-#    limits:
-#      cpu: 400m
-#      memory: 1024Mi
-#    requests:
-#      cpu: 400m
-#      memory: 1024Mi
+  resources:
+    limits:
+      cpu: 500m #400m
```

Comment:
Are the comments redundant? Innacurate? What do we mean with them?

Reply:
I will remove them. They are the original values